vuepress/docs/.vuepress/components/SpecialLayout.vue

```diff
@@ -69,7 +69,7 @@
             </div>
             <p>Download Helm chart</p>
             <div class="instruction">
-              curl -L -C - -O https://dev.entando.org/entando.yaml
+              curl -L -C - -O https://dev.entando.org/assets/yaml/entando.yaml
```

Comment:
We need to work towards not hosting these here. This will eventually get out of sync and we'll suffer for it.